test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -95,18 +95,32 @@ string ProtoConverter::visit(Literal const& _x)
 void ProtoConverter::consolidateVarDeclsInFunctionDef()
 {
 	m_currentFuncVars.clear();
+	yulAssert(!m_funcVars.empty(), "Proto fuzzer: Invalid operation");
+
 	auto const& scopes = m_funcVars.back();
 	for (auto const& s: scopes)
 		for (auto const& var: s)
 			m_currentFuncVars.push_back(&var);
+	yulAssert(!m_funcForLoopInitVars.empty(), "Proto fuzzer: Invalid operation");
+	auto const& forinitscopes = m_funcForLoopInitVars.back();
+	for (auto const& s: forinitscopes)
+		for (auto const& var: s)
+			m_currentFuncVars.push_back(&var);
```

Comment:
Consolidates all for-loop init vars declared inside current function that are alive and may hence be referenced.